### ISSUE TITLE
[Playlist] Synced playlists displaying default poster

### ISSUE THREAD

**Issue by benbou8231** [Kometa-Team/Kometa#1883]
### Version Number

1.20.0-nightly29

### What branch are you on?

nightly

### Describe the Bug

**Description:**

When using the `sync_to_users` template variable under `- pmm: playlist`, the playlists created and synced to the specified users do not retain the original playlist poster, instead displaying the default poster.

**Steps to Reproduce:**

1. Set the following configuration:

```
playlist_files:
- pmm: playlist
  template_variables:
    libraries: [library name]
    sync_to_users: Bob, Ben
```

2. Run the program.
3. Check the synced playlists for the non-admin users listed in `sync_to_users`.

**Expected Behavior:**

The synced playlists should retain the original playlist poster.

**Actual Behavior:**

The synced playlists display the default poster instead of the original one.

**Comment by benbou8231** [comment]
I tried a few things to update the poster of the synced playlist (in `sync_playlist()` from `builder.py`).
Everything give me a `(403) forbidden`.
Interestingly, the Plex log show me the same error (403) whenever I try with pmm or manually from Plex web interface.

So the feature is clearly not (yet?) supported in plex.

I found a [feature suggestion](https://forums.plex.tv/t/display-shared-playlist-more-prominently/734016) and a [reddit](https://www.reddit.com/r/PleX/comments/ovyxee/managed_users_playlist_posters/) post about this.

If the feature got supported at one point, here are the lines that were giving me the 403:

```python
    def sync_playlist(self):
        ...
        new_playlist = self.obj.copyToUser(user)
        new_playlist.uploadPoster(filepath=self.collection_poster.location)  # Unknown Error: (403) forbidden
        new_playlist.uploadPoster(url=self.obj.posterUrl)  # Unknown Error: (403) forbidden
        ...
```

Closing the issue (I just created it for visibility and possibly future implementation, put Plex is not known to be super fast to fix this kind of stuff).
